client/script/client.js

```diff
@@ -47,21 +52,37 @@ function validate(tables, callback) {
     }
   }
 
-  if(!questionsValid)
-    return callback(false, ADD_QUESTIONS_ERR);
+  if (errors.length !== 0) {
+    return callback(false, errors);
```

Comment:
Why check for errors.length at this point? Won't the return stop execution of the remaining error checks?